common/decisions/wc_debug_decision.txt

```diff
@@ -0,0 +1,56 @@
+﻿create_holy_orders_debug = {
+	picture = "gfx/interface/illustrations/decisions/decision_major_religion.dds"
+
+	is_shown = {
+		debug_only = yes
+	}
+
+	is_valid = {
+	}
+	effect = {
+		every_independent_ruler = {
+			add_to_list = holy_order_creators
+			
+			every_vassal = {
+				add_to_list = holy_order_creators
+			}
+		}
+		
+		# Creates 3 holy orders
+		every_in_list = {
+			list = holy_order_creators
+			
+			add_character_flag = { flag = holy_order_debug days = 1 }
+			
+			if = {
+				limit = {
+					can_execute_decision = create_holy_order_decision
+					
+					faith = {
+						trigger_if = {
+							limit = { exists = var:created_holy_orders }
+							var:created_holy_orders < 3
+						}
+						trigger_else = {
+							always = yes
+						}
+					}
+				}
+				
+				faith = {
+					if = {
+						limit = { exists = var:created_holy_orders }
+						change_variable = { name = created_holy_orders add = 1 }
+					}
+					else = {
+						set_variable = { name = created_holy_orders value = 1 days = 1 }
+					}
+				}
+				
+				execute_decision = create_holy_order_decision
+			}
+		}
+	}
+
+	ai_check_interval = 0
+}
```

Comment:
The button works and creates the number of holy orders, the problem comes with the name picking:
![image](https://user-images.githubusercontent.com/86002099/126046481-0c424e30-34df-4501-be71-a8644a0a7e19.png)

After testing a lot I can see there are some odd behaviours and the problem is... I'm not sure if it's your code or paradox', because you call create_holy_order_decision and it doesn't have that many changes so it should be ok. This is what I did:
- I tested with the holy light holy orders, I would click on the CREATE HOLY ORDERS decision and it would pic the first holy order and maybe the last holy order, then it would use some generic holy orders that wasn't specified there (somenthing like "Holy Light company of the Holy Light")
- After a while of trying and playing got tired and increased holy orders generated to 6 and did this:
![image](https://user-images.githubusercontent.com/86002099/126045871-627fa1d8-c06b-4fce-aa03-79951185c467.png)
 And the result would be something like:
![image](https://user-images.githubusercontent.com/86002099/126046286-fa28fc46-93b5-40cf-99e0-5a80dd010c20.png)

- With this:
![image](https://user-images.githubusercontent.com/86002099/126045890-3ed4f240-7476-43cf-a080-9e42ed995e93.png)
I got:
![image](https://user-images.githubusercontent.com/86002099/126046172-e47f4a1d-d5f8-4ea6-9162-9ac002cf6eb9.png)
- With holy orders ordered in the same way you get always the same holy orders in the game, also, after comparing I found out that it ignores the holy orders in a certain position and gets always the same holy orders. I think at some point it tries to create holy orders, can't, but the name is already used, somehow. The arrows indicate which holy orders get picked:
![image](https://user-images.githubusercontent.com/86002099/126046335-9b9a75c4-5cdf-455b-996a-276e25dea978.png)

Reply:
> but the name is already used, somehow.

Yes, I suppose you play the Holy Light faith and there are 2 branches. There's the dwarven branch of the Holy Light that takes these names.

Reply:
Yep, that was it, checked both Holy Light branches and all holy orders names are being used, LGTM!